src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -151,8 +151,8 @@ void EnMag_CheckSramResetCode(PlayState* play, EnMag* this) {
     s32 var_v1;
 
     var_v1 =
-        play->state.input[2].cur.button & (BTN_CRIGHT | BTN_CLEFT | BTN_CDOWN | BTN_CUP | BTN_R | BTN_L | BTN_DRIGHT |
-                                           BTN_DLEFT | BTN_DDOWN | BTN_DUP | BTN_START | BTN_Z | BTN_B | BTN_A);
+        play->state.input[2].cur.button & (BTN_A | BTN_B | BTN_Z | BTN_START | BTN_DUP | BTN_DDOWN | BTN_DLEFT | BTN_L |
+                                           BTN_R | BTN_DRIGHT | BTN_CUP | BTN_CDOWN | BTN_CLEFT | BTN_CRIGHT);
```

Comment:
did you intend to put `BTN_DRIGHT` where it is

Reply:
no, thank you